transportable-udfs-compile-utils/src/main/java/com/linkedin/transport/compile/UDFProperties.java

```diff
@@ -22,18 +23,27 @@
   private Multimap<String, String> _udfs;
 
   public UDFProperties() {
-    _udfs = HashMultimap.create();
+    _udfs = LinkedHashMultimap.create();
   }
 
-  void addUDF(String topLevelStdUDFClassName, String udfClassName) {
+  public void addUDF(String topLevelStdUDFClassName, String udfClassName) {
     _udfs.put(topLevelStdUDFClassName, udfClassName);
   }
 
+  public Multimap<String, String> getUdfs() {
```

Comment:
Please do not expose the map, which weakens the intended encapsulation of this class. One option is to expose getTopLevelClassName() and getTransportUdfClassNames(String topLevelClassName). Also suggestion: udfClassName --> transportUdfClassName, topLevelStdUDFClassName --> topLevelClassName.